source/git/source.go

```diff
@@ -539,6 +539,14 @@ func (gs *gitSourceHandler) Snapshot(ctx context.Context, g session.Group) (out
 		if err != nil {
 			return nil, err
 		}
+		_, err = checkoutGit.Run(ctx, "lfs", "install", "--local")
```

Comment:
Should we check if there are files being tracked by LFS first?

```
git lfs ls-files
```

Or maybe look for `.gitattributes` with `filter=lfs` like:

```scss
*.png filter=lfs diff=lfs merge=lfs -text
```

Reply:
Or maybe `git lfs status`?